src/_tests/integration/fire-cloud-utils.js

```diff
@@ -0,0 +1,33 @@
+const { findText, waitForNoSpinners, delay } = require('./integration-utils')
+
+
+const fireCloud = () => {
+  const openFireCloudSelector = async page => {
+    return (await page.waitForXPath(`//*[@data-test-id="workspace-selector"]`)).click()
```

Comment:
Either await or return, don't need both. This could be a synchronous function, like `findFirecloudClickable`.

Reply:
Similarly this one is awaiting the promise from `waitForXPath` and then the `return` is returning on the promise from the `click()` at the end of the line. `findFirecloudClickable` is only passing the promise it gets back from the `waitForXPath` function which is why there's no await/async on that one. So I believe it still needs both the `await` and `return` here?